File: lua/test/exists_test.lua
-- BluefinTecsMerchantServices SDK exists test

local sdk = require("bluefin-tecs-merchant-services_sdk")

describe("BluefinTecsMerchantServicesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# BluefinTecsMerchantServices SDK exists test

require "minitest/autorun"
require_relative "../BluefinTecsMerchantServices_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BluefinTecsMerchantServicesSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
